core/common/src/ViewProps.ts

```diff
@@ -26,6 +26,11 @@ export interface SectionDrawingViewProps {
   /** Transform from drawing coordinates to spatial coordinates. If undefined, use identity transform. */
   drawingToSpatialTransform?: TransformProps;
 }
+export interface SerializedViewStateProps {
```

Comment:
I'm a bit iffy calling this SerializedViewStateProps especially since there is an already existing ViewStateProps which has much more than what is contained in SerializedViewStateProps. Any suggestions for a name here?
SerializedViewStateData instead? something else?

Reply:
ViewCreator3dHelperData?

Reply:
We have a getViewStateData RPC operation which returns a ViewStateProps.
You've introduced getDefaultViewState3dData, which seems to introduce the concept of a "default view state", which is really more like a "manufactured (non-persistent)" view state.
Assuming we like the term "default" for this (I think we can do better), the method should probably return a DefaultViewState3dProps.

"Helper" is usually an unhelpful name; your helper looks like a DefaultViewState3dCreator to me.

Reply:
Could you explain more what you mean by "manufactured (non-persistent)"?

Would "initial" instead of "default" make any sense?

Reply:
getViewStateData returns data from a ViewDefinition element persisted in the iModel.
ViewCreator3d and the new rpc method manufacture a view out of thin air based on input parameters.
Perhaps a name like "CustomViewState" would convey this more clearly.